src/Authentication/Authentication/Cmdlets/InvokeMgGraphRequest.cs

```diff
@@ -416,60 +425,50 @@ private Uri PrepareUri(HttpClient httpClient, Uri uri)
             return uri;
         }
 
+        private void ThrowIfError(ErrorRecord error)
+        {
+            if (error != null)
+            {
+                WriteError(error);
+            }
+        }
         /// <summary>
         ///     Process Http Response
         /// </summary>
         /// <param name="response"></param>
-        internal void ProcessResponse(HttpResponseMessage response)
+        internal async Task ProcessResponseAsync(HttpResponseMessage response)
         {
             if (response == null) throw new ArgumentNullException(nameof(response));
-
-            var baseResponseStream = response.GetResponseStream();
-
+            var responseString = await response.Content.ReadAsStringAsync();
```

Comment:
This should be within `if (ShouldWriteToPipeline) {` block since it is only ever used when ShouldWriteToPipeline is true.